lib/screens/boards/boards_feed.dart

```diff
@@ -13,15 +13,53 @@ class BoardsFeed extends StatefulWidget {
 }
 
 class _BoardsFeedState extends State<BoardsFeed> {
+  final ScrollController _scrollController = ScrollController();
+  bool turnPage = false;
+
+  Future _onRefresh() {
+    context.read<Posts>().fetchPosts(widget.boardId);
+    return Future<void>.value();
+  }
+
+  Future addPosts() async {
+    if (turnPage)
+      Future.delayed(Duration.zero, () async {
+        int beforePostId = context.read<Posts>().posts.last.id;
+        await context.read<Posts>().addPosts(
+          boardId: widget.boardId,
+          beforePostId: beforePostId,
+        );
+      });
+  }
```

Comment:
async await pattern 이 의미없어 보입니다 ! 
```suggestion
  void addPosts() {
    if (turnPage)
      context.read<Posts>().addPosts(
          boardId: widget.boardId,
          beforePostId: beforePostId,
        );
      });
  }
```